src/frontend/src/optimizer/plan_node/stream_fs_fetch.rs

```diff
@@ -55,6 +55,7 @@ impl StreamFsFetch {
             source.catalog.as_ref().map_or(true, |s| s.append_only),
             false,
             FixedBitSet::with_capacity(source.column_catalog.len()),
+            Default::default(),
```

Comment:
maybe change it to `MonotonicityMap::default()` and leave a TODO comment with a github issue would be better. We do not need to finish it soon, but it should be noted as unfinished. Without the comment, It is difficult to discern whether the operators themselves cannot derive this property, or whether we have not considered them in detail.
Or have you already completed the derivation of the monotonicity for all operators 😮